doc/source/internals.rst

```diff
@@ -107,156 +108,7 @@ containers (``Index`` classes and ``Series``) we have the following convention:
 So, for example, ``Series[category]._values`` is a ``Categorical``, while
 ``Series[category]._ndarray_values`` is the underlying codes.
 
-
-.. _ref-subclassing-pandas:
```

Comment:
I suppose you kept the title (and reference to new location) for links to this place? In that case, I would also keep the reference label here (and rename it in extending.rst to 'extending-..' to be consistent with the others there. 
Because sphinx uses this label for making the url.

Reply:
> I suppose you kept the title (and reference to new location) for links to this place?

Yes, specifically for external links to http://pandas-docs.github.io/pandas-docs-travis/internals.html#subclassing-pandas-data-structures

I see what you mean now. Will update.